scripts/tokenization_query.py

```diff
@@ -0,0 +1,72 @@
+#
+# Pyserini: Reproducible IR research with sparse and dense representations
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Convert MSMARCO queries"""
+
+import argparse
+import multiprocessing
+from transformers import AutoTokenizer
+from tqdm import tqdm
+from scripts.ltr_msmarco.convert_common import get_retokenized
+"""
+add fields to query json with contents((BERT token)                                  
+"""
+
+
+parser = argparse.ArgumentParser(description='Convert MSMARCO-adhoc documents.')
+parser.add_argument('--input', metavar='input file', help='input file',
+                    type=str, required=True)
+parser.add_argument('--output', metavar='output file', help='output file',
+                    type=str, required=True)
+parser.add_argument('--proc_qty', metavar='# of processes', help='# of NLP processes to span',
```

Comment:
you don't need multiprocessing here, right?
simplify... fewer opportunities for bugs...